akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2PersistentClientSpec.scala

```diff
@@ -290,13 +290,13 @@ abstract class Http2PersistentClientSpec(tls: Boolean) extends AkkaSpecWithMater
       // need some demand on response side, otherwise, no requests will be pulled in
       client.responsesIn.request(1)
 
-      // first try is immedate
+      // first try is immediate
       probe.expectMsg("saw-reconnect")
-      probe.expectNoMessage(100.millis)
+      probe.expectNoMessage(250.millis) // 300 ms - 600 ms (using slightly lower values than expected to have some slack when expectation is run a little late)
       probe.expectMsg("saw-reconnect")
-      probe.expectNoMessage(200.millis)
+      probe.expectNoMessage(550.millis) // 600 ms - 1200 ms
       probe.expectMsg("saw-reconnect")
-      probe.expectNoMessage(500.millis)
+      probe.expectNoMessage(700.millis) // 750 ms - 1500 ms // (capped at 750ms (half of 1500 ms) - 1500ms)
```

Comment:
Hah, I had exactly these comments when working on it, where did those go 🤔

Reply:
Maybe you just saw them in the existing tests for the HTTP/1.1 client infrastructure? ;)